setup.py

```diff
@@ -1,15 +1,73 @@
 #!/usr/bin/env python
+import os
+import subprocess
+import sys
+from glob import glob
+from numpy import get_include as np_include
+from setuptools import setup, find_packages, Extension
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distribute_setup import use_setuptools
-    use_setuptools()
-    from setuptools import setup
+
+if os.path.exists('relic'):
+    sys.path.insert(1, 'relic')
+    import relic.release
+else:
+    try:
+        import relic.release
+    except ImportError:
+        try:
+            subprocess.check_call(['git', 'clone',
+                'https://github.com/jhunkeler/relic.git'])
```

Comment:
Let's not release code that goes to your personal repo.  Isn't there an SSB location to put relic?